recipes/pyproj/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
Feel free to put this in the meta.yaml directly. Obviously, at this stage, we still need the bld.bat though.